doc/cephfs/fuse.rst

```diff
@@ -109,23 +108,24 @@ For example::
        none    /mnt/mycephfs  fuse.ceph ceph.id=myuser,_netdev,defaults  0 0
        none    /mnt/mycephfs  fuse.ceph ceph.id=myuser,ceph.conf=/etc/ceph/foo.conf,_netdev,defaults  0 0
 
-Ensure you use the ID (e.g., ``admin``, not ``client.admin``). You can pass
+Ensure you use the ID (e.g., ``myuser``, not ``client.myuser``). You can pass
 any valid ``ceph-fuse`` option to the command line this way.
 
+To mount a subdirectory of the CephFS, add the following to ``/etc/fstab``::
+
+       none    /mnt/mycephfs  fuse.ceph ceph.id=myuser,ceph.client_mountpoint=/path/to/dir,_netdev,defaults  0 0
```

Comment:
Exactly what I wanted to be documented.

I would add a note, that any options from https://docs.ceph.com/docs/master/cephfs/client-config-ref/ can be specified.